components/channel_move_to_sub_menu/index.tsx

```diff
@@ -114,7 +116,8 @@ const ChannelMoveToSubMenu = (props: Props) => {
                     leadingElement={category.type === CategoryTypes.FAVORITES ? (<StarOutlineIcon size={18}/>) : (<FolderOutlineIcon size={18}/>)}
                     labels={text}
                     trailingElements={selectedCategory}
-                    onClick={(event) => handleMoveToCategory(event, category.id)}
+                    onKeyDown={(event) => handleMoveToCategory(event, category.id)}
+                    onMouseDown={(event) => handleMoveToCategory(event, category.id)}
```

Comment:
Why change this from `onClick` to `onKeyDown` and `onMouseDown`? We should probably make the `MenuItem` responsible for passing that handler as both props so that we don't require everyone who uses the menu to pass the same event handler twice

Reply:
i removed the two event handlers, MenuItem just have to handle onClick, and in the component library it will handle both events

Reply:
https://github.com/mattermost/mattermost-webapp/pull/12126/files#diff-cfbe7591ca3628ea663a185dffe7dba2ba36bc0b44cc10fc6655d581963ef65fR92-R97